spec/recipes/default_spec.rb

```diff
@@ -0,0 +1,384 @@
+require 'spec_helper'
+
+describe 'sc-mongodb::default' do
+  context 'with default node attributes' do
+    let(:mongod_conf_rhel) do
```

Comment:
Maybe a comment explaining why these aren't inline a in the individual tests? I try to keep unit tests as individually self contained as possible, so this means I've got to read two places to grok the test. 

If we can just extract the parts of tests that use this to only be in one place, would that balance the fragility and ease-of-jumping-into-tests?